### PR TITLE
Lang tet

### PR DESCRIPTION
**Pull request by PeteCoward**
@eraulo's work in a PR

### CODE REVIEW

File: num2words/lang_TET.py [hunk 1]
```diff
@@ -195,15 +195,15 @@ def to_year(self, val, longval=True):
         # Before changing this function remember this is used by pt-BR
         # so act accordingly
         if val < 0:
-            return self.to_cardinal(abs(val)) + ' antes de Cristo'
+            return self.to_cardinal(abs(val)) + ' antes Kristu'
         return self.to_cardinal(val)
 
-    def to_currency(self, val, currency='EUR', cents=True, separator=' e',
+    def to_currency(self, val, currency='EUR', cents=True, separator=' resin',
```
Comment (PeteCoward) [inline num2words/lang_TET.py]: we should not default to using Euros here I think

File: tests/test_tet.py [hunk 2]
```diff
@@ -80,379 +80,382 @@ def test_cardinal_integer(self):
...
+            num2words(1000000000000000000, lang='tet'),
+            'triliaun ida'
+        )
+        self.assertEqual(
+            num2words(2000000000000000000, lang='tet'),
+            'triliaun rua'
+        )
 
-    @skip("not implemented yet")
+    #@skip("not implemented yet")
```
Comment (PeteCoward) [inline tests/test_tet.py]: remove the skip code, not just comment out please

File: tests/test_tet.py [hunk 3]
```diff
@@ -290,68 +290,68 @@ def test_ordinal(self):
...
-        self.assertEqual(self.n2w.to_currency(13.00), 'sanulu resin tolu eurus')
-        self.assertEqual(self.n2w.to_currency(14.00), 'sanulu resin haat eurus')
-        self.assertEqual(self.n2w.to_currency(15.00), 'sanulu resin lima eurus')
-        self.assertEqual(self.n2w.to_currency(16.00), 'sanulu resin neen eurus')
-        self.assertEqual(self.n2w.to_currency(17.00), 'sanulu resin hitu eurus')
-        self.assertEqual(self.n2w.to_currency(18.00), 'sanulu resin walu eurus')
-        self.assertEqual(self.n2w.to_currency(19.00), 'sanulu resin sia eurus')
-        self.assertEqual(self.n2w.to_currency(20.00), 'ruanulu eurus')
+        self.assertEqual(self.n2w.to_currency(1.00), 'ida dólar')
+        self.assertEqual(self.n2w.to_currency(2.00), 'rua dólares')
```
Comment (PeteCoward) [inline tests/test_tet.py:294]: Is dólar and dólares really good Tetum?
I don't know the answer, but I have not seen it before.

Comment (eraulo) [reply tests/test_tet.py:294]: In Tetun we say **_Dolar ida_**

File: num2words/lang_TET.py [hunk 4]
```diff
@@ -22,7 +22,7 @@
 from .lang_EU import Num2Word_EU
 
 DOLLAR = ('dólar', 'dólares')
-CENTS = ('cêntimo', 'cêntimos')
+CENTS = ('cêntimu', 'cêntimus')
```
Comment (marobo) [inline num2words/lang_TET.py]: sentavu in tetun

File: num2words/lang_TET.py [hunk 5]
```diff
@@ -36,82 +36,82 @@ class Num2Word_TET(Num2Word_EU):
...
-                9: "nono",
+                1: "primeiru",
+                2: "segundu",
+                3: "terceiru",
+                4: "quartu",
+                5: "quintu",
+                6: "sextu",
+                7: "sétimu",
+                8: "oitavu",
+                9: "nonu",
```
Comment (marobo) [inline num2words/lang_TET.py]: 1: "dahuluk",
                2: "daruak",
                3: "datoluk",
                4: "dahaat",
                5: "dalimak",
                6: "daneen",
                7: "dahituk",
                8: "daualuk",
                9: "dasiak",

File: num2words/lang_TET.py [hunk 6]
```diff
@@ -36,82 +36,82 @@ class Num2Word_TET(Num2Word_EU):
...
+                1: "décimu",
+                2: "vigésimu",
+                3: "trigésimu",
+                4: "quadragésimu",
+                5: "quinquagésimu",
+                6: "sexagésimu",
+                7: "septuagésimu",
+                8: "octogésimu",
+                9: "nonagésimu",
             },
```
Comment (MarioAP) [inline num2words/lang_TET.py:86]: dasanuluk
daruanuluk
datolunuluk
dahaatnuluk
dalimanuluk
daneenuluk
dahitunuluk
dasianuluk

**Review by MarioAP — CHANGES_REQUESTED** [review]

**Pull request merged**